src/main/java/com/conveyal/r5/streets/EdgeStore.java

```diff
@@ -234,6 +234,9 @@ public EdgeStore (VertexStore vertexStore, StreetLayer layer, int initialSize) {
         // Comments below pasted from that document.
         // FIXME bicycle LTS should not really be flags, the categories are mutually exclusive and can be stored in 2 bits.
 
+        // If this flag is set, then its LTS has been loaded from OSM tags and we will not apply further processing to infer it.
+        BIKE_LTS_EXPLICIT (27),
```

Comment:
Worth a comment on why we skip to 27?

Reply:
This is just because the Bike LTS 1-4 were implemented as the highest 4 bit flags in the int. I just went one lower to group all LTS flags together. I suppose this would benefit from a comment but this is already released now.

Reply:
I added the comments and pushed directly to dev since this is just a pure documentation update.